SparkPost/support-docs
# WE-314 Add markdown Image component

**What Changed**
- Adds a script to copy images over to the `public` directory
- Adds the markdown `img` component

**How to verify**
- Run the site locally
- Visit a page with lots of images (eg: http://localhost:3000/momentum/3/3-reference/web-3-administration)
- Verify images render correctly
